src/http.js

```diff
@@ -0,0 +1,54 @@
+import { expandReferences } from "../";
+import axios from "axios";
+
```

Comment:
Is it overkill, or would a function called `request` which takes `params` but requires a `method` be useful here? all of the shortcuts we provide (`http.post`, `http.get`, etc.) could call _that_ function, but the generic `request` function could also be publicly available for use in expressions.